app.js

```diff
@@ -54,10 +55,12 @@ class App {
         const newConfIds = confIds.filter(id => !(id in this.conferenceStates));
         const removedConfIds = Object.keys(this.conferenceStates).filter(id => confIds.indexOf(id) === -1)
         newConfIds.forEach(newConfId => {
-            const dumpId = uuidv4();
+            const clientId = uuidv4();
```

Comment:
What's a good way to put an identified of the bridge instance in the identity? Prepend it to clientId?

Reply:
well if you want it in the actual file name that's stored in s3, prepending it would work, but if you just want it in the dump file itself, you could just add an additional field in the identity data object.